components/calendar/__tests__/__snapshots__/demo.test.js.snap

```diff
@@ -627,7 +627,7 @@ exports[`renders ./components/calendar/demo/basic.md correctly 1`] = `
               </div>
             </td>
             <td
-              class="ant-fullcalendar-cell"
+              class="ant-fullcalendar-cell ant-fullcalendar-today ant-fullcalendar-selected-day"
```

Comment:
That should not changed.

Reply:
just rebased however, snapshot test fails if this is not added.